### PR TITLE
1.8 pre-release changes

### PR DESCRIPTION
**Pull request by index-git**

### CODE REVIEW

File: src/layman/common/prime_db_schema/users.py [hunk 1]
```diff
@@ -62,6 +62,43 @@ def get_user_infos(username=None):
     return result
 
 
+def get_user_info_by_issuer_and_sub(issuer_id, sub):
```
Comment (index-git) [inline src/layman/common/prime_db_schema/users.py:65]: Sjednotit s get_user_infos

**Comment by jirik** [comment]
@index-git I've sqhuashed your last commit with 48bddd374e7f93020ed618672680508ba788b90f . Now it's ready to rebase.

**Review by jirik — APPROVED** [review]

**Comment by jirik** [comment]
@index-git I've just added one more commit https://github.com/jirik/layman/pull/186/commits/2230c5d3e27ecbd3be61548db9611c9073346a2f fixing https://github.com/jirik/layman/issues/187 . Without the change, newer version of Micka could not be used.

**Comment by index-git** [comment]
> @index-git I've sqhuashed your last commit with [48bddd3](https://github.com/jirik/layman/commit/48bddd374e7f93020ed618672680508ba788b90f) . Now it's ready to rebase.

I rewrote it. Fell free to squash it after review.

**Comment by jirik** [comment]
@index-git Thanks, I squashed it as it was.

**Pull request merged**